contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1027,7 +951,11 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         // behavior can be controlled. In particular, we enforce that flags are passed through
         // revert data as to retrieve execution metadata that would normally be reverted out of
         // existence.
-        (bool success, bytes memory returndata) = _target.call{gas: _gasLimit}(_data);
+
+        (bool success, bytes memory returndata) =
+            _isCreate
+            ? _handleExternalCreate(_gasLimit, _data, _contract)
```

Comment:
The basic idea is that `_handleExternalCreate` now wraps the creation result into a consistent `bool, data` return type which will have either `true, 0x` or `false, revert_data`.